app/scripts/modules/amazon/serverGroup/configure/serverGroupCommandBuilder.spec.js

```diff
@@ -26,6 +27,7 @@ describe('awsServerGroupCommandBuilder', function() {
 
     it('initializes to default values, setting usePreferredZone flag to true', function () {
       var command = null;
+      this.settings.providers.aws.defaults.iamRole = '{{application}}IAMRole';
```

Comment:
@zanthrash @ajordens i'm inclined to agree w/ you guys for now.  what do you think about switching to a Logback like syntax:  `Logger.log("{} i am in a {} mood", "today", "happy"); // today i am in a happy mood`?